maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -282,6 +288,22 @@ private Optional<PlexusConfiguration> createRulesFromCommandLineOptions() {
         return Optional.of(configuration);
     }
 
+    /**
+     * Filter our (remove) rules that have been specifically skipped via additional configuration.
+     *
+     * @param allRules list of enforcer rules to go through and filter
+     *
+     * @return list of filtered rules
+     */
+    private List<EnforcerRuleDesc> filterOutSkippedRules(List<EnforcerRuleDesc> allRules) {
+        if (rulesToSkip == null || rulesToSkip.length == 0) {
+            return allRules;
+        }
+        return allRules.stream()
+                .filter(ruleDesc -> !ArrayUtils.contains(rulesToSkip, ruleDesc.getName()))
```

Comment:
We can use `List<String>` as parameter type, then external library will be not needed 😄 
There aare more parameter as arrays ... we can improve in next PR

Reply:
Ah, I thought we need to use `String[]`, since it is also being used for `commandLineRules` Just tried though and it lists are working as well, which is definitely nicer.

I can create additional PR to get this cleaned up.

Reply:
https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Mapping_Collections_and_Arrays

Reply:
Here is a small follow up PR, replace the arrays with lists: https://github.com/apache/maven-enforcer/pull/210